CONTRIBUTING.md

```diff
@@ -80,6 +80,42 @@ git by going through these tutorials or a [Software
 Carpentry](https://software-carpentry.org/) workshop prior to making
 code contributions.
 
+### Virtual Environments
+
+Before you grab plasmapy from github, you are going to want to setup a sensible directory structure and a virtual environment. The virtual environment will allow you to import, run, and test your development version of plasmapy without contaminating or conflicting with other version of plasmapy or other packages that may be on your system.
```

Comment:
I'd suggest changing "plasmapy" to "PlasmaPy" and "github" to "GitHub" for consistency with the rest of the document.

Reply:
done